hw/bsp/rx/family.mk

```diff
@@ -22,11 +22,16 @@ endif
 
 ifeq ($(RX_NEWLIB),1)
   CFLAGS += -DSSIZE_MAX=__INT_MAX__
+  ifeq ($(LOGGER),rtt)
+    CFLAGS += -DCFG_TUSB_DEBUG_PRINTF=rx_rtt_printf
+    SRC_C += $(RTT_SRC)/RTT/SEGGER_RTT_printf.c
+  endif
 else
   # setup for optlib
   CFLAGS += -nostdinc \
     -isystem $(OPTLIBINC) \
-    -DLWIP_NO_INTTYPES_H
+    -DLWIP_NO_INTTYPES_H \
+    -DOPTLIB
```

Comment:
since optlib is deprecated now, can we  just remove it altogether and use NEWLIB since it is pretty standard on other ports.It is fine if you think we could still keep OPTLIB as an option.

Reply:
You're right, I'll remove it. I was thinking it was a pain to test with older version gcc. :smile: